### ISSUE TITLE
Problematic JPEG image

### ISSUE THREAD

**Issue by marcuslinke** [haraldk/TwelveMonkeys#22]
Hi Harald,

just found another image that can't be read with the JPEGImageReader. Maybe you will find some time to take a look. Not able to upload it here. Seems github has some problems with it also.

https://dl.dropboxusercontent.com/u/13086504/content.jpg

Cheers

**Comment by huxi** [comment]
Has this fix been released? I see a similar issue with 3.0 but didn't check 3.0.2 yet. Will do so now...

**Comment by huxi** [comment]
Alright... I got a NPE at
```
	com.twelvemonkeys.imageio.plugins.jpeg.JPEGImageReader.getSourceCSType(Unknown Source)
	com.twelvemonkeys.imageio.plugins.jpeg.JPEGImageReader.read(Unknown Source)
	javax.imageio.ImageReader.read(ImageReader.java:939)
```

But the reason for this was that the image with a jpeg extension was actually a png. Changing the extension the png fixed the problem. Should I file a new issue for that?

**Comment by haraldk** [comment]
@huxi My plugins does not do file format recognition by looking at file extensions, so I'm not sure how that happened... But yes, file an issue, with the code you are using and attach the image you used, and I'll look into it. 

The original issue has been fixed, at least both 3.0.1 and 3.0.2 has no problems reading the linked image.

-- 
Harald K
